src/components/navigation/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import { Affix, Button, Layout } from 'antd';
+import React from 'react';
+import { withRouter } from 'react-router';
+import { Link } from 'react-router-dom';
+
+const { Header } = Layout;
+
+@withRouter
+export default class NavBar extends React.PureComponent {
+  static propTypes = {};
+
+  render() {
+    if (this.props.location.pathname === '/') {
+      return null;
+    }
+
+    return (
+      <Affix>
```

Comment:
I should have put a note about what's going in here. It will eventually be breadcrumbs once I sort out some routing things. I'll file an issue about it.

Reply:
Ah, ok, makes sense ;)